stable/datadog/values.yaml

```diff
@@ -1,290 +1,256 @@
 
-  # The host of the Datadog intake server to send Agent data to, only set this option
-  # if you need the Agent to send data to a custom URL.
-  # Overrides the site setting defined in "site".
-  # dd_url: https://app.datadoghq.com
+  ## @param dd_url - string - optional - default: 'https://app.datadoghq.com'
+  ## The host of the Datadog intake server to send Agent data to, only set this option
+  ## if you need the Agent to send data to a custom URL.
+  ## Overrides the site setting defined in "site".
+  #
+  #  dd_url: https://app.datadoghq.com
 
+  ## @param logLevel - string - required
   ## Set logging verbosity.
   ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
-  ## Note: For Agent6 (image `datadog/agent`) the valid log levels are
+  ## Note: For Agent 6 (image `datadog/agent`) the valid log levels are
   ## trace, debug, info, warn, error, critical, and off
-  ##
+  #
   logLevel: INFO
 
+  ## @param nonLocalTraffic - boolean - optional
   ## Un-comment this to make each node accept non-local statsd traffic.
   ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
-  ##
-  # nonLocalTraffic: true
+  #
+  #  nonLocalTraffic: true
 
+  ## @param dogstatsdOriginDetection - boolean - optional
   ## Enable origin detection for container tagging
   ## https://docs.datadoghq.com/developers/dogstatsd/unix_socket/#using-origin-detection-for-container-tagging
-  ##
-  # dogstatsdOriginDetection: true
+  #
+  #  dogstatsdOriginDetection: true
 
+  ## @param useCriSocketVolume - boolean - required
   ## Enable container runtime socket volume mounting
+  #
   useCriSocketVolume: true
 
+  ## @param useDogStatsDSocketVolume - boolean - optional
   ## Enable dogstatsd over Unix Domain Socket
   ## ref: https://docs.datadoghq.com/developers/dogstatsd/unix_socket/
-  ##
-  # useDogStatsDSocketVolume: true
+  #
+  #  useDogStatsDSocketVolume: true
 
-  ## Set host tags.
-  ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
+  ## @param tags  - list of key:value elements - optional
+  ## List of tags to attach to every metric, event and service check collected by this Agent.
   ##
-  # tags:
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
 
-  ## Enables event collection from the kubernetes API
+  ## @param collectEvents - boolean - optional - default: false
+  ## Enables this to start event collection from the kubernetes API
   ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
-  ##
-  collectEvents: false
+  #
+  #  collectEvents: false
+
+  ## @param leaderElection - boolean - optional - default: false
+  ## Enables leader election mechanism for event collection.
+  #
+  #  leaderElection: false
 
-  ## Enables log collection
+  ## @param leaderLeaseDuration - integer - optional - default: 600
```

Comment:
Typo, the default value is 60 seconds: https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L263